bigtop-packages/src/common/trino/do-component-build

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+
+. `dirname $0`/bigtop.bom
+mkdir trino
+export trino_java_url=https://download.java.net/java/GA/jdk17.0.2/dfd4a8d0985749f896bed50d7138ee7f/8/GPL/openjdk-17.0.2_linux-x64_bin.tar.gz
```

Comment:
If additional JDK is needed, it must be installed by Puppet manifest under bigtop_toolchain. It should not be done everytime on packaging. Do not change the system default JDK in the manifest. Most of the products under Bigtop assuming JDK 8 now. See existing manifests for reference.
* https://github.com/apache/bigtop/blob/master/bigtop_toolchain/manifests/jdk.pp
* https://github.com/apache/bigtop/blob/master/bigtop_toolchain/manifests/jdk11.pp